acme_diags/plot/__init__.py

```diff
@@ -76,12 +76,12 @@ def get_colormap(colormap, parameters):
         cmap = LinearSegmentedColormap.from_list(name=colormap, colors=rgb_arr)
         return cmap
 
-    elif parameters.backend in ['vcs']:
-        n_levels = 240
-        cmap = LinearSegmentedColormap.from_list(name=colormap, colors=rgb_arr, N=n_levels)
-        vcs_cmap = matplotlib2vcs(cmap, vcs_name=colormap)
-
-        return vcs_cmap, list(range(n_levels))
+#    elif parameters.backend in ['vcs']:
+#        n_levels = 240
+#        cmap = LinearSegmentedColormap.from_list(name=colormap, colors=rgb_arr, N=n_levels)
+#        vcs_cmap = matplotlib2vcs(cmap, vcs_name=colormap)
+#
+#        return vcs_cmap, list(range(n_levels))
```

Comment:
In the files we keep, you comment out on the `vcs`-relevant code, but you've also deleted the entire directory `acme_diags/plot/vcs`. So, if we're sure we don't want to use `vcs` in the future, then I would say just delete these code blocks rather than commenting them out.